src/components/ScholarshipList.test.jsx

```diff
@@ -24,9 +31,15 @@ const fakeFirstPageOfManyResults = new Promise((resolve) => {
 });
 
 test('renders no results', async () => {
-  renderWithTheme(<ScholarshipList listFn={() => fakeNoResults} />, {
-    wrapper: MemoryRouter,
-  });
+  renderWithTheme(
+    <ScholarshipList
+      listFn={() => fakeNoResults}
+      noResultsNode={'no scholarships found'}
+    />,
```

Comment:
You can revert the rest of these changes now that the node is no longer needed. What you could do though is add a test case or two for the existence/presence of the property.